web-api/src/mind.rs

```diff
@@ -225,6 +277,43 @@ impl Default for StateConfig {
     }
 }
 
+// The results of iteration of the saturation benchmark
```

Comment:
:art: please note the difference between a double `//` for comments and a triple `///` for doc comments, only the tripple gets picked up for the doc generation. similarly for the other type doc comments below.
```suggestion
/// The results of iteration of the saturation benchmark
```

Reply:
Ah yes, I will change it, I must have skipped this one by accident